Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -1469,3 +1503,5 @@ theorem List.Sorted.lt_ord_of_lt [LinearOrder α] [IsWellOrder α (· < ·)] {l
       | head as => exact List.head_le_of_lt hmltl
       | tail b hi => exact le_of_lt (lt_of_lt_of_le (List.rel_of_sorted_cons hm _ hi)
           (List.head_le_of_lt hmltl))
+
+set_option linter.style.longFile 1700
```

Comment:
Is there no part of this file which could be split to a different one?

Reply:
I can see three such things. 
- That weird lemma on lists has no reason to be here (and I deprecate it in #16966). 
- `Cardinal.univ` also shouldn't be defined here (and it should be defined as `#Cardinal` instead). 
- Finally, addition and successors should be moved to `Arithmetic` (which will also be a few hundred lines shorter once I deprecate stuff like `bsup`, `lsub`, etc.). 
All of these will take a bit of time, though.

Reply:
Of course, the simpler alternative is to move the stuff about initial ordinals to `Cardinal/Ordinal`. But that file is very close to the 1.5k limit itself, so it'd be trading one thing for the other.

Reply:
What about the 200 lines porting note at the end of `Cardinal/Ordinal`? Regardless we should add a file for more advanced cardinal stuff soon, as `Cardinal/Basic` is bloated as well.

Reply:
Yeah, that porting note should be removed. Another idea I've had is splitting `Cardinal/Ordinal` into `Cardinal/Aleph` and `Cardinal/Arithmetic`.

Reply:
Actually, I've made the first of these splits in #16990. I'll move this definition to the `Aleph` file if the refactor goes through.